src/context.c

```diff
@@ -283,6 +283,7 @@ ly_ctx_new(const char *search_dir, uint16_t options, struct ly_ctx **new_ctx)
     struct ly_in *in = NULL;
     LY_ERR rc = LY_SUCCESS;
     struct lys_glob_unres unres = {0};
+    ly_bool no_advanced_builtin;
```

Comment:
Please rename this to be accurate. I would probably use a negation, `builtin_plugins_only`.

Reply:
done